Dockerfile

```diff
@@ -1,19 +1,17 @@
-FROM node:6-slim
+FROM node:10-slim
 
 WORKDIR /usr/src/app
 
-COPY package.json /usr/src/app
+COPY . /usr/src/app
 
 RUN apt-get update \
-    && apt-get install -y jq --no-install-recommends \
+    && apt-get install git -y \
```

Comment:
minor style (might also slightly increase speed of execution): you can simply catenate all packages to install `apt-get install -y git python ...`, also why `apt install build-essential` not `apt-get`?